src/hooks/useSort.ts

```diff
@@ -66,16 +62,16 @@ const useSort = () => {
 
     const sort = (features: IFeatureToggleListItem[]) => {
         switch (sortData.sortKey) {
-            case NAME:
+            case 'name':
```

Comment:
Constants still have utility with autocomplete and auto import capabilities when used at callsite.

Reply:
I don't understand this argument. Autocomplete works better with just TS, and you skip auto import. Constants auto import can prove misleading and hard to refactor in large apps.

![Untitled](https://user-images.githubusercontent.com/2625371/165453592-92e58fab-dabb-45cb-a506-d43dba058b68.png)

Reply:
You are right. Feel free to ignore the comments regarding these points.